keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/constraints/StatefulConstraintEvaluator.kt

```diff
@@ -24,10 +24,16 @@ import com.netflix.spinnaker.keel.api.events.ConstraintStateChanged
 /**
  * TODO: Docs.
  */
```

Comment:
stateful constraint evaluators now have another property to surface the type of the attributes, so that we can auto register it for jackson